packages/manager/apps/pci-instances/src/utils/index.ts

```diff
@@ -42,3 +45,7 @@ export const instancesQueryKey = (
   'instances',
   ...(rest && rest.length > 0 ? rest : []),
 ];
+
+export const mapAddressesToListItems = (
+  addresses?: DeepReadonly<TAddress[]>,
+): TListCellItem[] => addresses?.map(({ ip }) => ({ id: ip, name: ip })) ?? [];
```

Comment:
Not a big fan of having this in utils, this seems related to the datagrid component

Reply:
Moved to `mapper` folder